frontend/src/components/App/LoginForm.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component, PropTypes } from 'react'
+
+class LoginForm extends Component {
+  static propTypes = {
+    loginFormAuthToken: PropTypes.string
+  }
+
+  render () {
+    return <form className="loginAnywhere" id="new_user" action="/login" acceptCharset="UTF-8" method="post">
+      <input name="utf8" type="hidden" value="✓" />
+      <input type="hidden" name="authenticity_token" value="9z5D3vUGKM5ExKJ0CmhweE8qysvUqjFMwgMvbYXIlrnvg9sqJWIWgCt9lq28NZgyCaNudF+w+dRPD1pybeT4mg==" />
```

Comment:
haha, don't worry. this is very wip. I have to figure out how to handle forms in react. I've left this in for now so I remember to do that. Currently I've just commented out protect_from_forgery